Formula/scw.rb

```diff
@@ -16,20 +16,35 @@ class Scw < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GOBIN"] = buildpath
-    (buildpath/"src/github.com/scaleway/scaleway-cli").install Dir["*"]
+    system "go", "build", *std_go_args, "./cmd/scw"
 
-    system "go", "build", "-o", "#{bin}/scw", "-v", "-ldflags",
-           "-X github.com/scaleway/scaleway-cli/pkg/scwversion.GITCOMMIT=homebrew",
-           "github.com/scaleway/scaleway-cli/cmd/scw/"
+    output = Utils.safe_popen_read({ "SHELL" => "zsh" }, bin/"scw", "autocomplete", "script")
+    (zsh_completion/"_scw").write output
 
-    bash_completion.install "src/github.com/scaleway/scaleway-cli/contrib/completion/bash/scw.bash"
-    zsh_completion.install "src/github.com/scaleway/scaleway-cli/contrib/completion/zsh/_scw"
+    (bash_completion/"scw").write <<~EOS
```

Comment:
The zsh suggestion also applies to these completions.

Reply:
Fixed

Reply:
No, you reverted the zsh fix.

Reply:
The ZSH seems to be present here: https://github.com/Homebrew/homebrew-core/pull/57158/files#diff-f7f0cc8b2b5e67c0f2d2838506fa789aR21 Am I missing something?

Reply:
Yes, that removes the suggestion here: https://github.com/Homebrew/homebrew-core/pull/57158#discussion_r455852514

Reply:
Ok I've applied it to other suggestions